src/main/groovy/io/saagie/plugin/dataops/models/EnvVariable.groovy

```diff
@@ -0,0 +1,11 @@
+package io.saagie.plugin.dataops.models
+
+class EnvVariable {
+	
+	String scope
+	
+	ArrayList<String> name
```

Comment:
why an array list of string is called "name" ?

Reply:
The wording are requested by the client 
![image](https://user-images.githubusercontent.com/57661415/90026143-d8539d00-dcae-11ea-9509-f86e959134b3.png)